ldms/rules.mk

```diff
@@ -0,0 +1,24 @@
+# 1. Fix duplicate titles when converting to man
+# 	- Remove "NAME" and "=====" underneath
+# 	- Add :title: to first sentence after :Date:
+# 2. Remove :ref:`<>` links in .rst to avoid rs2man "Unkown interpretd text role "ref"" errors
```

Comment:
Could we adopt the formatting used in the example:

https://docutils.sourceforge.io/sandbox/manpage-writer/rst2man.txt

and avoid this too?